docs/nodes/nodes-library/trigger-nodes/GitlabTrigger/README.md

```diff
@@ -0,0 +1,30 @@
+---
+permalink: /nodes/n8n-nodes-base.gitlabTrigger
+---
+
+# GitLab Trigger
+
+[GitLab](https://gitlab.com/) is a web-based DevOps lifecycle tool that provides a Git-repository manager providing wiki, issue-tracking and continuous integration/continuous deployment pipeline features, using an open-source license, developed by GitLab Inc.
+
+::: tip 🔑 Credentials
+You can find authentication information for this node [here](../../../credentials/Gitlab/README.md).
+:::
+
+
+## Example Usage
+
+This workflow allows you to receive updates for Gitlab events. You can also find the [workflow](https://n8n.io/workflows/528) on the website. This example usage workflow would use the following node.
```

Comment:
Can you please make sure that it's correctly written as 'GitLab' everywhere in this document?